### PR TITLE
Annual growth effects (into correct branch)

### PR DESCRIPTION
**Pull request by jkikstra**
See #51 for discussion.

### CODE REVIEW

File: src/components/MarketDamagesBurke_annual.jl [hunk 1]
```diff
@@ -138,7 +138,7 @@ end
     igdp_ImpactatActualGDPperCap_ann = Variable(index=[year, region])
 
     isat_ImpactinclSaturationandAdaptation= Variable(index=[time,region])
-    isat_ImpactinclSaturationandAdaptation_ann = Variable(index=[year, region], unit = "\$")
+    isat_ImpactinclSaturationandAdaptation_ann = Variable(index=[year, region])
```
Comment (jrising) [inline src/components/MarketDamagesBurke_annual.jl:141]: Why did you remove this?

**Review by jrising — APPROVED** [review]
Seems fine.
By the way, is there a reason why the annual variables are in the same components as the timestep variables? Why not have separate components for these that just do the annual calculations, and have these added in after each time step component when one wants annual calcs?

**Comment by jkikstra** [comment]
> 
> 
> Seems fine.
> By the way, is there a reason why the annual variables are in the same components as the timestep variables? Why not have separate components for these that just do the annual calculations, and have these added in after each time step component when one wants annual calcs?

That seems like another good way to do it. It would in principle be more neat indeed if you consider the annualisation as an 'add-on'. However, if you would now quickly want to just take one (annual) version of the model, that is easier to do in the current setup. Also, adding components does come with extra lines in the 'main.jl' file, as you need to pass on more variables->parameters.

File: src/components/MarketDamagesBurke_annual.jl (hunk 1 above)
Comment (jkikstra) [reply src/components/MarketDamagesBurke_annual.jl:141]: Ah no specific reason actually, I did a quick check for whether the units were similar in the timestep version and the annual version, that's why I had deleted it.
